.circleci/config.yml

```diff
@@ -114,7 +114,7 @@ commands:
       - run:
           name: Install Node.js
           command: |
-            curl -sL https://deb.nodesource.com/setup_14.x | bash -
+            curl -sL https://deb.nodesource.com/setup_20.x | bash -
```

Comment:
The consumer of node is serverless, I think. I assume it plays nice with this node version?

Reply:
It has in other pipelines, so I assume so